server/app.coffee

```diff
@@ -1,2 +1,8 @@
 Meteor.startup ->
-    
+    config =
+        run_seed: process.env.SEED == "true"
+
+    if config.run_seed
+        Meteor.call 'seedPosts'
+        Meteor.call 'seedTopics'
+        Meteor.call 'seedSections'
```

Comment:
Poprawiłem, teraz na produkcji powinien być automatyczny seed
Ale uwaga! meteor --production nie uruchamia apki jak na produkcji, to raczej symulacja, bo wtedy i tak zmienna NODE_ENV jest ustawiona na "development"
Tu jest napisane, że żeby było "production" to trzeba zrobić bundle i odpalić przez node: https://github.com/meteor/meteor/issues/3839